apps/masterbots.ai/lib/helpers/ai-helpers.ts

```
@@ -0,0 +1,373 @@
+      return payload
+    case 'Anthropic':
+      return payload.map(
+        (message, index) =>
+          ({
+            role: index
+              ? message.role.replace('system', 'assistant')
+              : message.role.replace('system', 'user'),
+            content: message.content
+          }) as MessageParam
+      )
+    case 'OpenAI':
```

Biome

[error] Useless case clause.

because the default clause is present:

Unsafe fix: Remove the useless case.


(lint/complexity/noUselessSwitchCase)